didcomm/did_doc/did_doc.py

```diff
@@ -68,6 +68,16 @@ def didcomm_services(self) -> List[DIDCommService]:
         """
         pass
 
+    @abstractmethod
+    def get_verification_method(self, id: DID_URL) -> VerificationMethod:
```

Comment:
We should either return this method in all implementations, or don't return it at all.